app/styles/base/_helpers.scss

```diff
@@ -326,3 +326,7 @@ a.disabled {
 .pointer-events-none {
   pointer-events: none;
 }
+
+.no-invalid-box:invalid {
```

Comment:
This should go in _forms.scss and apply to all inputs instead of this particular one. There are probably others and firefox is inconsistent with every other browser.